common/shlibs

```diff
@@ -3998,3 +3998,4 @@ libsonic.so.0 libsonic-0.2.0_1
 libtickit.so.3 libtickit-0.4.1_1
 libnotcurses.so.2 notcurses-2.0.4_1
 libnotcurses++.so.2 notcurses-2.0.4_1
+libantilib.so.1 libantilib-3.1.2_1
```

Comment:
`libantimicrox-3.1.2_1`, not `libantilib-3.1.2_1`

Reply:
Oops, typo!